R/prep_data.R

```diff
@@ -62,7 +62,8 @@
 #' @param scale Logical. If TRUE, numeric columns will be scaled to have a
 #'   standard deviation of 1. Default is FALSE.
 #' @param make_dummies Logical. If TRUE (default), dummy columns will be created
-#'   for categorical variables.
+#'   for categorical variables. When TRUE, nominal characters vector columns
```

Comment:
I don't think users will see any change from this, right? Dummy columns (integers) will still be created; the characters/factors are intermediates that I think are invisible to the user.

Reply:
You are right. I was trying to be thorough, but this doesn't matter.